hack/common.sh

```diff
@@ -86,9 +86,9 @@ OS_TYPE=$(uname)
 
 # Override the image name when this is invoked from openshift ci
 if [ -n "$OPENSHIFT_BUILD_NAMESPACE" ]; then
-	CATALOG_FULL_IMAGE_NAME="registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:${CATALOG_IMAGE_NAME}"
+	CATALOG_FULL_IMAGE_NAME="registry.build02.ci.openshift.org//${OPENSHIFT_BUILD_NAMESPACE}/stable:${CATALOG_IMAGE_NAME}"
```

Comment:
There was an email that we shouldn't use hardcoded URLs like this. Since it keeps changing, jobs will mostly fail.
We need to make sure we do the right thing here.

Reply:
this has been changed